### DISCUSSION TITLE
better-npm-audit audit audit ?

### DISCUSSION THREAD

**Discussion by freddieduffield** [jeemok/better-npm-audit#52] [Q&A] [answered]
Hello 👋  

Thank you for `better-npm-audit`, it is a great utility! 

In the readme, in the section for packaging scripts it states:
```json
{
  "scripts": {
    "prepush": "npm run test && npm run audit",
    "audit": "better-npm-audit audit audit"
  }
}
```
the `audit` script has `audit audit` following `better-npm-audit`. I am wondering is there a reason for this double audit? It seems to work fine with a single `audit`, but just wanted to check if is required and if so what does it do?

Many thanks!

**Comment by jeemok** [comment, marked as answer]
ah, it was a mistake! Thanks for pointing it out! 👍🏻

**Reply by freddieduffield** [reply]
Thanks, much appreciated 👍
